shortfin/python/shortfin_apps/flux/components/builders.py

```diff
@@ -70,9 +70,7 @@ def get_params_filenames(model_params: ModelParams, model=None, splat: bool = Fa
     else:
         for idx, mod in enumerate(modnames):
             # schnell and dev weights are the same, except for sampler
-            subtype = (
-                "schnell" if model_params.is_schnell and mod == "sampler" else "dev"
-            )
+            subtype = "schnell" if model_params.is_schnell else "dev"
```

Comment:
For flux, it would be nice to not have multiple copies of the same artifact. Dev/schnell have everything the same other than the transformer.

Maybe I'm wrong, is there a good reason to duplicate the artifacts--in case of a change upstream or downstream?

Reply:
Now they are the same. I made the always prefix with just `flux_`. The export+upload would have to be refactored anyway.